.github/workflows/integration-tests-vm-type.yml

```diff
@@ -94,11 +94,15 @@ jobs:
           GCP_SSH_KEY: ${{ secrets.GCP_SSH_KEY }}
           GCP_SSH_KEY_PUB: ${{ secrets.GCP_SSH_KEY_PUB }}
 
-      - name: Normalize job ID
+      - name: Set environment
         run: |
           NORM_JOB_ID="${JOB_ID/-/}"
           echo "JOB_ID=${NORM_JOB_ID}" | tee -a "$GITHUB_ENV"
 
+          if [[ '${{ contains(github.event.pull_request.labels.*.name, 'integration-tests-trace-logging') }}' == 'true' ]]; then
+            echo "COLLECTOR_LOG_LEVEL=trace" | tee -a "$GITHUB_ENV"
```

Comment:
[ultra-nit] Don't know if it's necessary to `tee` the environment variable, the following steps will print it out if you expand them